api/core/span_context.go

```diff
@@ -62,6 +62,12 @@ func (t TraceID) MarshalJSON() ([]byte, error) {
 	return json.Marshal(hex.EncodeToString(t[:]))
```

Comment:
It would be good to have this defer to `t.String()`.

```suggestion
	return json.Marshal(t.String())
```

Reply:
Done.